manifests/stage-multus-cni/0050-multus-ds.yml

```diff
@@ -36,6 +36,12 @@ spec:
           args:
             - "--cni-version=0.4.0"
             - "--multus-conf-file={{- if .CrSpec.Config -}}/tmp/multus-conf/00-multus.conf{{- else -}}auto{{- end -}}"
```

Comment:
nit: it would be good to mention why we use mount configmap to /tmp/multus-conf/00-multus.conf  here and remove other file

Reply:
so I think @Mmduh-483  change will work only when --multus-conf-file=auto. 
@Mmduh-483  have you check the flow if .CrSpec.Config ?

Reply:
@moshe010 actually, my comment doesn't relate to this change but having lines ##38 and ##44 together looks a bit confusing without some code investigation. That's why I suggested adding a commennt

Reply:
@moshe010 it is working either use auto or custom config `00-multus.conf` is going to be generated

Reply:
@e0ne `/tmp/multus-conf/00-multus.conf` is where the configMap mounted in the container

Reply:
So just explain that in a comment

Reply:
Added comment for it